client/webpack.common.config.js

```diff
@@ -6,12 +6,8 @@ module.exports = {
 
   // the project dir
   context: __dirname,
-  entry: ['./assets/javascripts/App'],
+  entry: ['jquery', 'jquery-ujs', './assets/javascripts/App'],
```

Comment:
It can be less ugly this way:

```js
entry: {
  app: './assets/javascripts/App',
  vendor: ['jquery', 'jquery-ujs']
}
```

But since all chunks will be squashed by assets pipeline, this will be useful only for the code appearance.